exercises/fizz-buzz/example.R

```diff
@@ -0,0 +1,22 @@
+fizz_buzz <- function(input) {
+  condition <- function(i){
+    if(i %% 3 == 0 && i %% 5 == 0){
+      return("Fizz Buzz")
+    }else if(i %% 3 == 0){
+      return("Fizz")
+    }else if(i %% 5 == 0){
+      return("Buzz")
+    }else{
+      return(i)
+    }
+  }
+  
+  out <- sapply(seq(1, input), condition)
+  return (cat(out, sep = "\n"))
+}
+
+
+
+
+
+
```

Comment:
This white-space is superfluous. Please apply [`styler::style_dir("fizz-buzz")`](https://github.com/r-lib/styler#styler) to remove it, and some other linting issues.